app/controllers/review_mapping_controller.rb

```diff
@@ -165,11 +130,54 @@ def check_outstanding_reviews?(assignment, reviewer)
         @num_reviews_completed += 1 if !map.response.empty? && map.response.last.is_submitted
       end
       @num_reviews_in_progress = @num_reviews_total - @num_reviews_completed
-      @num_reviews_in_progress < Assignment.max_outstanding_reviews
+      # if all reviews done => no outstanding reviews - bug fix
+      @num_reviews_in_progress > 0 && @num_reviews_in_progress <= Assignment.max_outstanding_reviews
```

Comment:
Comment Ameya: 
Edit: `@num_reviews_in_progress < Assignment.max_outstanding_reviews`

Reply:
@chiragbheemaiah to see if tests pass after this(newly added tests)